cloudformation/media-atoms-dynamo-code.json

```diff
@@ -15,8 +15,12 @@
             "Description": "Code audit dynamo table",
             "Type": "String",
             "Default": "audit-media-atom-maker-CODE"
+        },
+        "UploadTrackingAtomMakerCodeTable": {
```

Comment:
Chatted IRL and we've agreed to leave it as-is but document the use of two cloudformations for the stack